source/includes/steps-fle-convert-to-a-remote-master-key-azure.yaml

```diff
@@ -230,6 +242,39 @@ content: |
            To use the Azure Key Vault, you must use `pymongocrypt <https://pypi.org/project/pymongocrypt/>`__
            version 1.1 or later in your application's environment.
 
+     .. tab::
+        :tabid: csharp
+
+        .. code-block:: csharp
+
+           var keyVaultClient = new MongoClient(_connectionString);
+           var clientEncryptionOptions = new ClientEncryptionOptions(
+               keyVaultClient: keyVaultClient,
+               keyVaultNamespace: _keyVaultNamespace,
+               kmsProviders: kmsProviders);
+
+           var clientEncryption = new ClientEncryption(clientEncryptionOptions);
+           var azureKeyName = Environment.GetEnvironmentVariable("FLE_AZURE_KEY_NAME");
+           var azureKeyVaultEndpoint = Environment.GetEnvironmentVariable("FLE_AZURE_KEYVAULT_ENDPOINT"); // typically <azureKeyName>.vault.azure.net
+           var azureKeyVersion = Environment.GetEnvironmentVariable("FLE_AZURE_KEY_VERSION"); // Optional
+           var dataKeyOptions = new DataKeyOptions(
+               masterKey: new BsonDocument
+               {
+                   { "keyName", azureKeyName },
+                   { "keyVaultEndpoint", azureKeyVaultEndpoint },
+                   { "keyVersion", () => azureKeyVersion, azureKeyVersion != null }
+               });
+
+           var dataKeyId = clientEncryption.CreateDataKey("azure", dataKeyOptions, CancellationToken.None);
+           Console.WriteLine($"Azure DataKeyId [UUID]: {dataKeyId}");
+           var dataKeyIdBase64 = Convert.ToBase64String(GuidConverter.ToBytes(dataKeyId, GuidRepresentation.Standard));
+           Console.WriteLine($"Azure DataKeyId [base64]: {dataKeyIdBase64}");
+
+        .. note::
+
+           To use the AWS Key Vault, you must use
```

Comment:
Nitpick: I wonder if we can avoid "To use..., ... use ...".
How about "To access the AWS Key Vault, you must use...

Reply:
I think this is good and will address it when I do the GCP ticket.